src/components/ListedSpecies/SexSection.jsx

```diff
@@ -0,0 +1,56 @@
+import {
+  FormControl,
+  FormLabel,
+  Grid,
+  GridItem,
+  NumberDecrementStepper,
+  NumberIncrementStepper,
+  NumberInput,
+  NumberInputField,
+  NumberInputStepper,
+} from '@chakra-ui/react';
+import React from 'react';
+import { useFormContext } from 'react-hook-form';
+import CollapsibleSection from '../CollapsibleSection/CollapsibleSection';
+
+const inputs = [
+  '# of Male Adults',
+  '# of Male Fledges',
+  '# of Male Chicks',
+  '# of Female Adults',
+  '# of Female Fledges',
+  '# of Female Chicks',
+];
+
+const SexSection = () => {
+  const { setValue, getValues } = useFormContext();
+
+  return (
+    <CollapsibleSection title="Sex">
+      <Grid templateColumns="repeat(3, 1fr)" gap="2em">
+        {inputs.map((input, idx) => (
+          <GridItem key={input}>
+            <FormControl>
+              <FormLabel>
+                {input}
+                <NumberInput
+                  min={0}
+                  onChange={val => setValue(`sex[${idx}]`, parseInt(val, 10))}
+                  defaultValue={getValues().sex[idx]}
+                >
+                  <NumberInputField />
+                  <NumberInputStepper>
+                    <NumberIncrementStepper />
+                    <NumberDecrementStepper />
+                  </NumberInputStepper>
+                </NumberInput>
```

Comment:
Should total input for sex be limited to the # of that type of bird they input in general information?

Like if they said they saw 5 adults, they should be unable to add more male adult birds if female+male = 5.

Reply:
added